app/controllers/supplies_controller.rb

```diff
@@ -1,18 +1,22 @@
 class SuppliesController < ApplicationController
   before_action :set_supply, only: [:show, :edit, :update]
-  
+
   def index
+    authorize @supplies
```

Comment:
authorize @supplies should be updated to use policy scope, ie: @supplies = policy_scope(Supply)